packages/advanced-logic/specs/payment-network-btc-address-based-0.1.0.md

```diff
@@ -29,6 +21,12 @@ As a payment network, this extension allows to deduce a payment `balance` for th
 
 Note: to use the bitcoin testnet just replace the id by "pn-testnet-bitcoin-address-based"
 
+## Interpretation
+
+Any bitcoin transaction reaching the address `addPaymentAddress` is considered as payment.
```

Comment:
(mistake in the original docs I think)
isn't it `paymentAddress`, instead of `addPaymentAddress`? The latter is the name of the method I believe
Same below.